data/avgi/map planets.txt

```diff
@@ -0,0 +1,539 @@
+	description "The fact that there is more gas in the torus than one gas giant can account for has not been unnoticed, however, and many among the Avgi believe the entire system to be an artificial construct engineered by the long dead Predecessors."
+	government Uninhabited
+	security 0
+
+
+planet Vanikos
+	attributes ocean uninhabited
+	landscape land/beach18
+	description "Vanikos is an uninhabited cold watery moon. That is all. WIP"
+	government Uninhabited
+
+
+planet Kalakil
+	attributes forest uninhabited
+	landscape land/mountain38
+	description "Kalakil is an uninhabited misty forest moon. That is all. WIP"
+	government Uninhabited
+
+planet Janalos
+	attributes forest uninhabited
+	landscape land/forest14
+	description "Janalos is an uninhabited lush forest planet covered in insects. That is all. WIP"
+	government Uninhabited
+
+
+planet Yiddoi
+	attributes uninhabited
+	landscape land/mountain41
+	description "Yiddoi appears to be the product of a failed attempt at terraforming predating the Avgi, or one that was abandoned before it was complete. A blend of clearly synthetic reflective compounds was at one point injected into the moon's atmosphere, ostensibly to reduce incoming sunlight and cool its boiled surface."
+	government Uninhabited
+
+
+planet Katradol
+	attributes frozen research uninhabited
+	landscape land/snow30
+	description `Katradol is a large rocky world, what human scientists would call a "super-Earth." Far from the local primary, Katradol is extremely cold; during the winter its atmosphere condenses in the higher latitudes, where it snows frozen nitrogen. Many of Katradol's mountains are made mostly of ice, which is strong enough in the extreme cold to stretch upwards for kilometers, even under the high gravity. The roots of these mountains generated some interest in the Avgi scientific community due to the many exotic phases of ice that form under the intense pressures, but Katradol's deep gravity well made ice mining uncompetitive with comet harvesting in the region.`
+	government Uninhabited
+
+
+planet "Alonis Observatory"
+	attributes avgi research station telescope uninhabited
+	landscape land/station56
+	description "Decades ago, this large observatory station was constructed in order to peer into the anomalous region known as the Ember Waste. Before the Avgi were able to harden their ships against the prevalent ion storms, this was their only way to extract its secrets. An array of advanced telescopes covering the entire electromagnetic spectrum sits side by side with more exotic equipment, from gravitational wave interferometers to neutrino detectors."
+	description "After Avgi starships advanced to the point of being able to reliably withstand ion storms, it was trained deeper into the Ember Waste, looking further back in both space and time. It gathered an impressive amount of data before the Incursion forced its abandonment, though fortunately the telescope seems to have been left unscathed."
```

Comment:
This specifically refers to the part of the Ember Waste near Headwind, yeah? Just to make a note of it.